gapic-generator/test/gapic/presenters/service_presenter_test.rb

```diff
@@ -69,4 +69,12 @@ def test_deduped_resources_with_duplicated_patterns
     assert_equal 1, patterns.size
     assert_equal "projects/*/locations/*", patterns[0].pattern_template
   end
+
+  def test_deprecated_service
+    presenter = service_presenter :garbage, 'DeprecatedService'
```

Comment:
Double quotes here and again a few lines below.

Reply:
Done.